src/sentry/utils/cloudfunctions.py

```diff
@@ -50,7 +51,7 @@ def upload_function_files(client, code, env_variables):
 
     upload_url = client.generate_upload_url(
         request=GenerateUploadUrlRequest(
-            parent="projects/hackweek-sentry-functions/locations/us-central1"
+            parent="projects/" + SENTRY_FUNCTIONS_PROJECT_NAME + "/locations/us-central1"
```

Comment:
@vpandiarajan20 actually can you make the region a setting as well? We can leave the default as `us-central1` though